protocols/cosi/cosi.go

```diff
@@ -234,18 +221,8 @@ func (pc *ProtocolCosi) handleCommitment(in *CosiCommitment) error {
 	return pc.SendTo(pc.Parent(), outMsg)
 }
 
-type ChallengeHook func(*CosiChallenge) error
-
 // StartChallenge start the challenge phase. Typically called by the Root ;)
 func (pc *ProtocolCosi) StartChallenge() error {
-	// first check the hook
-	/*if pc.challengeHook != nil {*/
-	//return pc.challengeHook(nil)
-	/*}*/
-
-	if pc.Message == nil {
-		return fmt.Errorf("%s StartChallenge() called without message (=%v)", pc.Node.Name(), pc.Message)
-	}
```

Comment:
Why remove that check ?
If we start a CoSi round without signing anything, then the user of the CoSi must surely do something wrong and should be alerted of that ...